src/components/AllServicesDropdown/AllServicesMenu.tsx

```diff
@@ -68,8 +68,8 @@ const AllServicesMenu = ({ setIsOpen, isOpen, menuRef, linkSections, favoritedSe
       <Backdrop>
         <Panel variant="raised" className="pf-u-p-0 chr-c-panel-services-nav">
           <PanelMain>
-            <Sidebar>
-              <SidebarPanel>
+            <Sidebar style={{ height: '698px', overflow: 'auto' }}>
```

Comment:
I think we might have to compute the height value based on the height of the screen. Otherwise, we will get into some double scrollbar weirdness.

![Screenshot from 2023-05-23 09-09-04](https://github.com/RedHatInsights/insights-chrome/assets/22619452/5988de82-2166-4873-b97e-0aa413651eb5)

Reply:
@Hyperkid123 I'm only seeing the double scrollbar when the viewport height is less than 850px and it's a non-issue in mobile. Is this an edge case?